src/globals/js/feature-flags.js

```diff
@@ -59,7 +59,7 @@
  * }
  */
 
-exports.breakingChangesX = false;
-exports.componentsX = false;
-exports.grid = false;
-exports.gridColumns16 = false;
+exports.breakingChangesX = true;
+exports.componentsX = true;
+exports.grid = true;
+exports.gridColumns16 = true;
```

Comment:
```suggestion
exports.gridColumns16 = false;
```

Default should be false to help with interop, this should go to true in v11, I think 👍

Reply:
OK cool thanks @joshblack for letting me know!